visual_regression_tracker/visualRegressionTracker.py

```diff
@@ -16,14 +17,16 @@ class VisualRegressionTracker:
     projectId: str = None
     headers: dict = None
 
-    def __init__(self, config: Config):
+    def __init__(self, config: Config = None):
         """
         Creates a new VisualRegressionTracker
 
         :param config: The configuration to use.
         """
-        self.config = config
+        self.config = config or Config.default()
         self.headers = {'apiKey': self.config.apiKey}
+        print(self.config, config)
```

Comment:
config printed twise in case explicit code config was used

Reply:
removed

Reply:
actually I found this info really usefull in order to see real config that was used 
gonna think of adding some info logs like we have for Java sdk